handle/Objects.lua

```diff
@@ -0,0 +1,2 @@
+require('CeresStdLib.handle.Timer')
```

Comment:
This entire file is unnecessary, at least while there's nothing general for every handle.